storybook/styles/docs.css

```diff
@@ -393,15 +393,17 @@
     border: 2px solid var(--o-ui-b-alias-low-break);
     border-top-color: var(--o-ui-white);
     animation: o-ui-sb-spinner 1s linear infinite;
-    transform: translate(-50%, -50%) rotate(0deg);
+    translate: -50% -50%;
+    /* might be possible to remove to validate */
```

Comment:
So, is it possible? :)

Reply:
It was! Will remove unnecessary rotate declaration.